src/pages/components/NavContainer/NavContainer.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from "react";
+import Nav from "./Nav/Nav";
+import MyCurator from "./MyCurator/MyCurator";
+import SearchBox from "./SearchBox/SearchBox";
+import "./NavContainer.scss";
+
+class NavContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      userName: null,
```

Comment:
String으로 들어올 데이터라면 아래와 같이 표현해주시면 더 좋을것 같네요! 
```suggestion
      userName: ””,
```

Reply:
오! string 처음알았어요~ 수정할게요!

Reply:
@Link717 따옴표가 약간 옆으로 누워보이는건 기분탓이겠죠...? 👀

Reply:
"" 게 작성했는데 옆으로 눕혔네욥....👀